public/app/features/alerting/unified/utils/alertmanager.ts

```diff
@@ -95,7 +95,7 @@ export function parseMatcher(matcher: string): Matcher {
   };
 }
 
-export function parseMatchers(matcherQueryString: string): Matcher[] {
+export function parseMatchers(matcherQueryString = ''): Matcher[] {
```

Comment:
nit, but I wouldn't provide a default, would leave this to the caller. Stronger type safety, prevents you from accidentally forgetting to provide an argument or providing `undefined` when you didn't mean to